js/weather-api/index.html

```diff
@@ -7,6 +7,17 @@
 </head>
 
 <body>
+    <select id="selectCountry">
+        <option value="london" selected>ロンドン</option>
+        <option value="tokyo">東京</option>
+        <option value="pali">パリ</option>
+        <option value="new%20york">ニューヨーク</option>
```

Comment:
現状でも動いていますがこう書いた方が適切ですね。人間が読みやすいので。
```suggestion
        <option value="new york city">ニューヨーク</option>
```